data/catalogs/deltares_data.yml

```diff
@@ -1175,7 +1191,12 @@ hydro_lakes:
     category: surface water
     notes: HydroLAKES.v10_extract
     source_author: Arjen Haag
```

Comment:
This is actually just the hydrolakes database so I would update the source info to match hydrolakes including the author: https://www.hydrosheds.org/products/hydrolakes 
Then for processing_notes: extract of hydrolakes to only include natural lakes and exclude reservoirs and control lakes (ie filter on Lake_type = 1)